app/models/planning_application.rb

```diff
@@ -215,6 +215,14 @@ def full_address
     "#{address_1}, #{town}, #{postcode}"
   end
 
+  def secure_change_url(local_authority, application_id, secure_token)
```

Comment:
I think this should be a helper method, rather than a method on the model?

Reply:
I've changed my mind about this actually. In the future we might want this to be set via the API, so it being on the model makes sense.